app/scrapers/__init__.py

```diff
@@ -42,6 +42,8 @@ def feed_gen(query, engine, count=10, qtype=''):
     engine = old_names.get(engine, engine)
     if engine in ('quora', 'youtube'):
         urls = scrapers[engine].search_without_count(query)
+    elif engine in ('baidu',) and qtype == 'news':
```

Comment:
Why use `in ('baidu',)`
You can use `if engine == 'baidu':`
Since there is only one option to choose from

Reply:
Mistakenly I approved it.